brcmf_bus_ops.txdata() uses a paramater of type struct sk_buff which
was not declared here when compiling against kernele 4.8 and older.

--- a/drivers/net/wireless/broadcom/brcm80211/brcmfmac/bus.h
+++ b/drivers/net/wireless/broadcom/brcm80211/brcmfmac/bus.h
@@ -6,6 +6,7 @@
 #ifndef BRCMFMAC_BUS_H
 #define BRCMFMAC_BUS_H
 
+#include <linux/skbuff.h>
 #include "debug.h"
 
 /* IDs of the 6 default common rings of msgbuf protocol */
